docs/src/running-tests-js.md

```diff
@@ -2,112 +2,145 @@
 
 **You will learn**
 
 - [How to run tests from the command line](/running-tests.md#command-line)
 - [How to debug tests](/running-tests.md#debugging-tests)
 - [How to open the HTML test reporter](/running-tests.md#test-reports)
 
+## VS Code extension
 
-## Run tests in UI Mode
+Tests can be run right from VS Code using the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-playwright.playwright). Once installed you can simply click the green triangle next to the test you want to run or run all tests from the testing sidebar. Check out our [Getting Started with VS Code](./getting-started-vscode.md#running-tests) guide for more details.
 
-Run your tests with [UI Mode](./test-ui-mode.md) for a better developer experience with time travel debugging, watch mode and more.
+![Playwright VS Code extension](https://github.com/microsoft/playwright/assets/13063165/47726e70-683b-4bd5-94de-7d03dd45c30f)
 
-  ```bash
-  npx playwright test --ui
-  ```
+## Command line
 
-## Command Line
+You can run your tests with the `playwright test` command. This will run your tests on all browsers as configured in the `playwright.config` file. Tests run in headless mode by default meaning no browser window will be opened while running the tests and results will be seen in the terminal.
 
-- Running all tests
+```bash
+npx playwright test
+```
+
+![tests running in command line](https://github.com/microsoft/playwright/assets/13063165/981c1b2b-dc7e-4b85-b241-272b44da6628)
+
+### Run tests in UI mode
+
+We highly recommend running your tests with [UI Mode](./test-ui-mode.md) for a better developer experience where you can easily walk through each step of the test and visually see what was happening during, before and after each step. UI mode also comes with many other features such as the locator picker, watch mode and more.
 
-  ```bash
-  npx playwright test
-  ```
+```bash
+npx playwright test --ui
+```
+
+![UI Mode](https://github.com/microsoft/playwright/assets/13063165/c5b501cc-4f5d-485a-87cc-66044c651786)
 
-- Running a single test file
+Check out or [detailed guide on UI Mode](./test-ui-mode.md) to learn more about it's features.
 
-  ```bash
-  npx playwright test landing-page.spec.ts
-  ```
+### Run tests in headed mode
 
-- Run a set of test files
+To run your tests in headed mode use the `--headed` flag. This will open up a browser window while running your tests and once finished the browser window will close.
```

Comment:
```suggestion
To run your tests in headed mode use the `--headed` flag. This will give you the ability to visually see, how Playwright interacts with the website.
```